README.org

```diff
@@ -27,6 +27,8 @@ the 28.0 development tree.  See issue [[https://github.com/sharplispers/log4cl/i
 
 ** Enabling Log4Slime
 
+To enable Log4Sly run the following in your Lisp:
```

Comment:
Should this say "Log4Slime"?

Reply:
Yes, stupid copy and paste error.  Fixed.